app/uk/gov/hmrc/voapropertylinking/binders/clients/GetClientsParameters.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.voapropertylinking.binders.clients
+
+import java.time.LocalDate
+
+import binders.{Params, ValidationResult}
+import uk.gov.hmrc.voapropertylinking.binders.validation.ValidatingBinder
+
+case class GetClientsParameters(
+      name: Option[String] = None,
+      appointedFromDate: Option[LocalDate] = None,
+      appointedToDate: Option[LocalDate] = None)
```

Comment:
Do we use these dates at all? Couldn't see in the PR or the other when they get used.

Reply:
there is currently no way for the frontend to submit the dates at the moment. I kept the dates here because the modernised endpoint can take dates. If we ever decide to add date filtering to the frontend, we will only need to update the frontend